packages/chart/src/CdcChart.jsx

```diff
@@ -989,15 +987,30 @@ export default function CdcChart({ configUrl, config: configObj, isEditor = fals
       })
     }
 
+    if (columnObj === undefined) {
```

Comment:
This section is related to problems with naming between config.dataFormat which is for y axis data values and for Additional columns.  I tried not to just rename things in case that breaks old charts.  This code section maps over so that the code below for adding prefix, suffix, rounding, commas works...